etc/config/llvm.defaults.properties

```diff
@@ -1,6 +1,6 @@
 binaryHideFuncRe=^(__.*|_(init|start|fini)|(de)?register_tm_clones|call_gmon_start|frame_dummy|(_GLOBAL__sub_I_|\.plt).*|.*@plt(-0x[0-9a-f]+)?)$
 objdumper=objdump
-demangler=c++filt
+demangler=/opt/compiler-explorer/clang-trunk/bin/llvm-cxxfilt
```

Comment:
the default should not contain any reference to `/opt/compiler-explorer` , `llvm-cxxfilt` might be good enough, but I don't know enough about clang installations to know if there will be a `/usr/bin/llvm-cxxfilt` . I have 1 example system here that has it installed but as `/usr/bin/llvm-cxxfilt-12` and no non-versioned link.

Reply:
I think we should be good to go on the clang installations
```
/opt/compiler-explorer/clang+llvm-16.0.5-cross-hexagon-unknown-linux-musl/x86_64-linux-gnu/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-4.0.0/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-4.0.1/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-5.0.0/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-5.0.1/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-5.0.2/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-6.0.0/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-6.0.1/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-7.0.0/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-7.0.1/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-7.1.0/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-8.0.0/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-8.0.1/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-9.0.0/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-9.0.1/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-10.0.0/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-10.0.1/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-11.0.0/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-11.0.1/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-12.0.0/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-12.0.1/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-13.0.0/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-13.0.1/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-14.0.0/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-15.0.0/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-16.0.0/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-17.0.1/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-assertions-4.0.0/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-assertions-5.0.0/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-assertions-5.0.1/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-assertions-5.0.2/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-assertions-6.0.0/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-assertions-6.0.1/bin/llvm-cxxfilt
/opt/compiler-explorer/clang-assertions-7.0.0/bin/llvm-cxxfilt
```

Reply:
yes but the .defaults properties are used by local installations

Reply:
Yeah on my system it's `/usr/bin/llvm-cxxfilt-14` too. So probably the best I can do is restore `c++filt`, which sadly _would_ have some surprising side effects for local deployments.

Reply:
> yes but the .defaults properties are used by local installations

oh my bad, didn't see this was defaults.properties :)

Reply:
Fixed per @partouf 's advice